src/components/pages/Menu.tsx

```diff
@@ -0,0 +1,1000 @@
+import React, {useEffect} from 'react';
```

Comment:
Yeah that's what I didn't know I ended up storing all states and everything in the 'EditMenu" page so that way their weren't stored in their own components and passed through props. I am not sure if there is a better way to do this but it definitely got a lot longer when the accordion was put in because now there had to be more arrays and a lot of state switching between the different accordions. I am hesitant to change it now since we need to focus on other things, but knowing what I know now I could definitely clean this up a lot. When I started writing this I had never used react or typescript before.